src/Common/src/Interop/Windows/kernel32/Interop.CreateNamedPipeClient.Uap.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
how is this file any different than the existing Interop.UnsafeCreatefile.Uap.cs ? can we use that

Reply:
The problem is this one gets out as SafePipelineHandle, at first try I can't cast it from the UnsafeCreateFile.Uap - it was my first try, let me know if there is a way to do that.

Reply:
Uh, yeah I guess this is the best I can think of.